index.html

```diff
@@ -0,0 +1,79 @@
+            <h1 id="splash-title">ROCKET MAN</h1>
+            <div id="splash-title-rocket">
+                <img src="img/fire-png-gif-489.gif" id="splash-flames">
+                <img src="img/rocket-large.png" id="splash-rocket-img">
+            </div>
+        </div>
+        <div id="splash-open-game">Play Me</div>
+        <div id="splash-bottom-wrapper">
+            <img src="img/new-york-large.png" id="splash-city">
+            <div id="splash-instructions">
+                <p>A megalomaniac has successfully researched and developed nuclear warheads, he plans to rain down fire and
+                    fury on the major cities of the world. It's your job to stop him!</p>
+                <ul>
+                    <li>Click on the start button to enter the game screen, then push the red button to begin the defence
+                        of planet Earth.</li>
+                    <li>Click on the cities targeted by nuclear warheads to enable their defence mechanisms.</li>
+                    <li>You can also use keys: Q, W, E and R on your keyboard to save the corresponding cities.</li>
+                    <li>You gain points by successfully deflecting missiles. However, if you fail to do this then watch the
+                        death toll skyrocket!</li>
+                    <li>The game will get progressively harder, can you keep pace?</li>
+                </ul>
+            </div>
+        </div>
+    </div>
+    <main>
+        <div class="game-wrapper">
+            <div class="col">
+                <img src="img/missile-drop-down-white.png" class="missile" id="bomb-1">
+                <div id="city-1" class="city city1" data-city="1"></div>
+            </div>
+            <div class="col">
+                <img src="img/missile-drop-down-white.png" class="missile" id="bomb-2">
+                <div id="city-2" class="city city2" data-city="2"></div>
+            </div>
+            <div class="col">
+                <img src="img/missile-drop-down-white.png" class="missile" id="bomb-3">
+                <div id="city-3" class="city city3" data-city="3"></div>
+            </div>
+            <div class="col">
+                <img src="img/missile-drop-down-white.png" class="missile" id="bomb-4">
+                <div id="city-4" class="city city4" data-city="4"></div>
+            </div>
+        </div>
+        <div class="keyboard-wrapper">
+            <div class="keyboard">Q</div>
+            <div class="keyboard">W</div>
+            <div class="keyboard">E</div>
+            <div class="keyboard">R</div>
+        </div>
+        <div class="score-wrapper">
+            <div class="score totals">
+                <h2>DEFLECTS:</h2>
+                <p id="score">0</p>
+            </div>
+            <div class="toll totals">
+                <h2>DEATH TOLL:</h2>
+                <p id="toll">1,125,423</p>
+            </div>
+        </div>
+        <div id="start" class="start">START</div>
```

Comment:
why is this here? start what? its set to display: none; please remove

Reply:
fixed in de56b2c